osbs/api.py

```diff
@@ -680,7 +691,17 @@ def _do_create_prod_build(self, git_uri, git_ref,
         )
         build_request.set_openshift_required_version(self.os_conf.get_openshift_required_version())
         build_request.set_repo_info(repo_info)
-        if build_request.scratch:
+
+        builds_for_koji_task = []
+        if koji_task_id and not is_auto:
```

Comment:
I think we want to do this just for orchestrator builds. If we do that, there's no need to check `is_auto` since orchestrator autorebuilds will not go through this code path.

Reply:
ok